packages/app/src/update/actions/start-update-os/start-update-os.action.ts

```diff
@@ -62,6 +67,17 @@ export const startUpdateOs = createAsyncThunk<
 
     for (const release of releases) {
       state = getState() as RootState & ReduxRootState
```

Comment:
About this state, wondering if we could deal with that type of getState once and not cast it every time we use it?

Reply:
The process is asynchronous so it is a reason, but I think that we could refactor it when time allows us to do.